creator-node/src/services/stateMachineManager/stateMachineConstants.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
   // Max number of completed/failed jobs to keep in redis for the state monitoring queue
   RECONCILIATION_QUEUE_HISTORY: 300,
 
+  // Max number of completed/failed jobs to keep in redis for the state monitoring queue
+  MANUAL_SYNC_QUEUE_HISTORY: 300,
```

Comment:
I see you have `QUEUE_HISTORY.MANUAL_SYNC` below so this constant should be necessary